Remotion/Data/DomainObjects.Web.Test/Queries2.xml

```diff
@@ -17,7 +17,7 @@
  ! along with re-motion; if not, see http://www.gnu.org/licenses.
 -->
 <queries
-    xmlns="http://www.re-motion.org/Data/DomainObjects/Queries/2.0" 
+    xmlns="http://www.re-motion.org/Data/DomainObjects/Queries/2.0"
```

Comment:
strictly speaking, this change is irrelevant. it's also helpful and there's enough noise that it won't matter.